### PR TITLE
test: create default wallet in extended tests

### PR DESCRIPTION
**Pull request by Sjors**
This was omitted from #15454

### CODE REVIEW

**Comment by jnewbery** [comment]
Can these be fixed by adding the `-wallet=` command line argument, like the other tests in #15454?

**Comment by promag** [comment]
@jnewbery if we ditch automatic wallet creation then we could already avoid that approach.

**Comment by Sjors** [comment]
@jnewbery done

**Comment by jnewbery** [comment]
utACK 165b8fd561

Thanks for the quick fix!

File: test/functional/feature_dbcrash.py [hunk 1]
```diff
@@ -75,7 +75,8 @@ def skip_test_if_missing_module(self):
     def setup_network(self):
         self.add_nodes(self.num_nodes, extra_args=self.extra_args)
         self.start_nodes()
-        self.import_deterministic_coinbase_privkeys()
+        for n in self.nodes:
+            self.import_deterministic_coinbase_privkeys()
```
Comment (ryanofsky) [inline test/functional/feature_dbcrash.py:79]: In commit "test: create default wallet in extended tests" (165b8fd5617e4bd4d5f99586306a66f084d95d13)

Could you add a comment here? It's unclear why you'd want to call the same function n times.

**Review by ryanofsky — APPROVED** [review]
Code review ACK 165b8fd5617e4bd4d5f99586306a66f084d95d13

**Comment by ryanofsky** [comment]
Previous version 9df44e0b97ba74b4df01e600b4c19e9d75310498 also looked good, but John's suggestion to use -wallet="" is good because it provides more consistency with other tests so they be cleaned up easier later. There are a few ways tests could conceivably create default wallets:

1. Not pass any -wallet arguments or call createwallet RPC, just rely on bitcoind default wallet creation (only possible before #15454)
2. Pass ["-wallet="] argument and rely on test framework to strip out the argument and turn it into a createwallet RPC call behind the scenes
3. Just call createwallet RPC directly from the test
4. Give test framework a `wants_default_wallet` or similar test setup option, so tests have a more direct way of saying whether they want a default wallet or not and we can get rid of the "-wallet=" interception stuff.

#15454 moved us from (1) to (2). Originally this PR used approach (3) for extended tests, now it uses (2). Probably we want to end up doing (4)

File: test/functional/feature_dbcrash.py (hunk 1 above)
Comment (Sjors) [reply test/functional/feature_dbcrash.py:79]: That's a leftover bug from my refactor.

**Comment by Sjors** [comment]
Third time's a charm.

**Review by ryanofsky — APPROVED** [review]
Code review ACK a5f5374b43275ce4529c3a44c4b44e478d35e079. Just reverted a leftover diff since last review

**Review by glozow — COMMENTED** [review]
utACK https://github.com/bitcoin/bitcoin/pull/19971/commits/a5f5374b43275ce4529c3a44c4b44e478d35e079

**Pull request merged**